apps/root/src/state/transactions/transactionUpdater.tsx

```diff
@@ -123,43 +126,111 @@ export default function Updater(): null {
 
   const parseTxExtendedTypeData = useCallback(async (tx: TransactionDetails) => {
     let extendedTypeData = {};
-
     if (!tx.receipt) {
       return extendedTypeData;
     }
 
-    if (tx.type === TransactionTypes.newPair) {
-      extendedTypeData = {
-        id: toHex(tx.receipt.logs[tx.receipt.logs.length - 1].data),
-      };
-    }
-
-    if (tx.type === TransactionTypes.newPosition) {
-      const parsedLog = await transactionService.parseLog({
-        logs: tx.receipt.logs,
-        chainId: tx.chainId,
-        eventToSearch: 'Deposited',
-      });
+    const getTokenWithPrice = async (token: Token) => {
+      const tokenBasePrice = await priceService.getUsdHistoricPrice([token], undefined, token.chainId);
+      const tokenPrice = parseUsdPrice(token, 10n ** BigInt(token.decimals), tokenBasePrice[token.address]);
+      return { ...token, price: tokenPrice };
+    };
 
-      if ('positionId' in parsedLog.args) {
+    switch (tx.type) {
```

Comment:
This should all go inside a try catch, so as to not block that we confired a transaction if fetching the price fails